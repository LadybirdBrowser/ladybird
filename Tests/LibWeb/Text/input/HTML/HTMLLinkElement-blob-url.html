<script src="../include.js"></script>
<div id="test-element">Test Element</div>
<script>
    asyncTest((done) => {
        const cssContent = `
            #test-element {
                color: red;
                font-weight: bold;
            }
        `;
        const blob = new Blob([cssContent], { type: "text/css" });
        const blobURL = URL.createObjectURL(blob);

        const link = document.createElement('link');
        link.rel = 'stylesheet';
        link.href = blobURL;
        document.head.appendChild(link);

        link.onload = () => {
            const testElement = document.getElementById('test-element');
            const computedStyle = window.getComputedStyle(testElement);
            println(computedStyle.color);
            println(computedStyle.fontWeight);
            done();
        };
    })
</script>
