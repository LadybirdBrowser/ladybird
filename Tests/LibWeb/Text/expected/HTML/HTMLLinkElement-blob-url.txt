rgb(255, 0, 0)
700
